postgres/pgdump.go

```diff
@@ -342,6 +359,15 @@ func processCopyStmt(conv *internal.Conv, n nodes.CopyStmt) *copyOrInsert {
 	} else {
 		logStmtError(conv, n, fmt.Errorf("relation is nil"))
 	}
+	if _, ok := conv.SrcSchema[table]; !ok {
```

Comment:
Suggest a small tweak in the comment:

		// If we don't have schema information for a table, we drop all copy
		// statements for it. The most likely reason we don't have schema information
		// for a table is that it is an inherited table - we skip all inherited tables.

Reply:
updated comment